bin/superuser

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+# shellcheck source=bin/_config.sh
+source "$(dirname "${BASH_SOURCE[0]}")/_config.sh"
+
+_django_manage createsuperuser "$@"
```

Comment:
Is this shortcut really required and not too much specific as the manage script almost handles it?

Reply:
This shortcut already saved precious seconds of my life :smile: :trollface:

Reply:
You lazy boy :trollface: 

![](https://media3.giphy.com/media/8hXXilmk33wtmAGJNu/giphy.gif)